chart/templates/server-deployment.yaml

```diff
@@ -120,6 +120,13 @@ spec:
                   key: {{ $secret.name }}
           {{- end }}
           {{- end }}
+          {{- range $key, $path := .Values.parameters }}
```

Comment:
I had to add root to this line to be able to build.
`{{- range $key, $path := $root.Values.parameters }}
`
To ensure this version will not break dev, I will copy the values.yml from the [dev manifest values](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/dev/values.yaml) then try to build the helm charts.

Reply:
fixed